rustls/src/client/tls13.rs

```diff
@@ -72,6 +73,8 @@ pub(super) fn handle_server_hello(
     hello: ClientHelloDetails,
     our_key_share: Box<dyn ActiveKeyExchange>,
     mut sent_tls13_fake_ccs: bool,
+    ech_state: Option<EchState>,
+    server_hello_msg: &Message,
```

Comment:
Style nit: feels like `server_hello_msg` should come earlier in this list? At least before `ech_state`, which seems like it would be longer lived.

Reply:
Done.